routers/web/repo/issue.go

```diff
@@ -760,6 +760,14 @@ func setTemplateIfExists(ctx *context.Context, ctxDataKey string, possibleFiles
 		ctx.Data[ctxDataKey] = template.Content
 
 		if template.Type() == api.IssueTemplateTypeYaml {
+			// Replace field default values by values from query
+			for _, field := range template.Fields {
+				field_value := ctx.FormString(field.ID)
+				if field_value != "" {
+					field.Attributes["value"] = field_value
```

Comment:
Also, this potentially allows for cross site scripting attacks by inserting something like
`…/issues/new?textField=<script>alert('You've been hacked);</script>` into the URL.
Or do we ensure that it cannot happen due to go templates security aspect?

Reply:
I don't think we should worry about this:

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/9418365/215376733-46d2d566-8bf2-4b2f-83b9-c643d115a13e.png">

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/9418365/215376965-2fdd1ff2-ada7-4915-921e-72b5b6eb9bab.png">
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/9418365/215377243-b59fe585-e12e-4903-a2bb-5357778fb578.png">
(So many "so what", it's for dramatic effect only, I'm a polite guy.😁)